authors/apps/profiles/tests/test_profiles.py

```diff
@@ -76,12 +76,3 @@ def test_successfully_listing_profiles(self):
         response = self.list_profiles()
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertIsInstance(response.data.get("profiles")[0], dict)
-
-    def test_listing_profiles_with_unauthorised_user(self):
```

Comment:
Hi @Bakley ,
Instead of deleting this test, I think you could update it to show the success on retrieval of data without authentication. This would ascertain that the GET profiles endpoint gives success data for both authenticated and unauthenticated users

Reply:
@Ogutu-Brian Authenticated users who are not verified can't access any GET endpoint.